src/WebInterface/Infrastructure/Security/UserProvider.php

```diff
@@ -24,6 +46,24 @@ public function supportsClass(string $class): bool
 
     public function loadUserByIdentifier(string $identifier): UserInterface
     {
-        return new User($identifier);
+        try {
+            $user = $this->identityQueryBus->handle(
```

Comment:
This goes against the current convention that everything is going through `Gaming\WebInterface\Application\IdentityService`. However, this layer of indirection isn't needed anyways, because the `Bus` interface can also be used to route request either via network or in process calls, achieving the same goals. Will probably create an issue to delete that.